### ISSUE TITLE
Allow usage with an API key instead of a password

### ISSUE THREAD

**Issue by ghost** [amishyn/confluence-api-client#5]
Allow usage with an API key instead of a password. If you can already do that, that's great but an example in your README.md would be helpful.

**Comment by cpb** [comment]
You can use [personal access token](https://confluence.atlassian.com/enterprise/using-personal-access-tokens-1026032365.html) in place of a password. It also looks like [API tokens can be used in place of passwords too](https://support.atlassian.com/atlassian-account/docs/manage-api-tokens-for-your-atlassian-account/).

IE: ![Atlassian shows in a screenshot that you can use API tokens in](https://confluence.atlassian.com/cloud/files/938839638/938839639/1/1507010022324/Screen+Shot+2017-09-25+at+5.09.09+pm.png)

@leewsb would you be willing to summarize some of this in a PR on the README?

**Comment by ChrisAllen1982** [comment]
I don't believe this is strictly true. It depends on Confluence Atuthentication settings. If Basic authentication is not enabled (Bearer only), then you cannot use the method suggested.

My suggestions is to add a second constructor to allow token based authentication. @amishyn I am happy to extend to do so. I will create a pull request. Is that acceptable?
